### PR TITLE
NOSTORY: This is a stand alone VPC created only with default security…

### PR DESCRIPTION
**Pull request by patrickmukumbu**
… group, nacl and route table

@JohnNMwangi  After reading through documentation, both for AWS and terraform, as well as having a look at a previous private module for VPC I saw it would be good to have the different networking resources have their own modules.

C.C @hugoalmeida264

### CODE REVIEW

File: examples/main.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module "aws_vpc" {
+  source               = "./.."
+  cidr_block           = "172.16.0.0/16"
```
Comment (j0hnmwangi) [inline examples/main.tf]: Is it possible to have another example with CIDR block from AWS IPAM?

**Review by j0hnmwangi — CHANGES_REQUESTED** [review]
Hey Patrick, what your thought on the suggestion below ?

File: examples/main.tf (hunk 1 above)
Comment (patrickmukumbu) [reply examples/main.tf]: Yes, just a moment I add that @JohnNMwangi

**Review by j0hnmwangi — APPROVED** [review]
great module & examples.

**Pull request merged**